topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md

```diff
@@ -0,0 +1,226 @@
+>    >
+>    > This is an incredibly useful feature as it saves you from forgetting to destroy a VM. Especially for GPU nodes it can help you ensure that they disappear after your computation is complete.
+>    {:.tip}
+>
+>    ![almost there! some final details reads this page, asking for name and description. both have been filled out with 'pulsar'. a yellow submit button waits at the bottom](images/surf6.png)
+>
+> 7. Click submit when you are happy with the configuration.
+>
+{: .hands_on}
+
+Once done, the workspace will be created for you. You'll need to wait ~5 minutes usually. Go for a beverage ☕️
+
+![workspace list showing a workspace named pulsar being created.](images/surf7.png)
+
+## Using Pulsar on SRC
+
+> <hands-on-title>Access the Information Page</hands-on-title>
+> 
+> 1. Once the workspace is up, you'll see an **Access** link:
+> 
+>    ![A small plus button is hovered over which says Add. Below galaxy-test is shown running with a yellow Access button](images/surf3.png)
+> 
+> 2. Click that will show you a Pulsar information page. This page is running on your pulsar node itself, and is restricted to ensure only authorised members can access the contents. It includes some configuration you will need to copy to your Galaxy node in order to make use of the Pulsar node.
+> 
+>    ![pulsar configuration information page showing an about with admins and metadata like workspace fqdn. Configuration for galaxy is shown below including XML changes](images/surf8.png)
+{: .hands_on}
+
+This information page should have more than enough information to connect this Pulsar instance to your Galaxy server. You will need to reference information from this page in the following steps:
+
+> <hands-on-title>Configuring Galaxy to use SRC Pulsar</hands-on-title>
+> 1. Collect the requirements for accessing the Galaxy machine. You will need:
+> 
+>    - your username from the first step
+>    - your SSH key that is associated with your SRAM account
+> 
+> 2. SSH into your *Galaxy* machine (not pulsar!).
+> 
+>    ```
+>    ssh -i path/to/your/sram-ssh-key username@galaxy.src-winter-school.src.surf-hosted.nl
+>    ```
+> 
+> 3. You will need to `sudo su` to do anything useful. Do that now.
+> 4. `cd /srv/galaxy/` to move into the directory Galaxy configuration is stored.
+> 5. The configuration is discussed fully in the Pulsar information, but it will be briefly covered here as well. Generally there are a few steps that must be followed:
+>
+>    - A runner must be registered
+>    - A destination/environment must be added with the pulsar details
+>    - Some tools should be redirected to this Pulsar
+>
+>    Here is an example of what those changes *might* look like in your Galaxy node. ([FAQ: how to read a diff]({% link topics/admin/faqs/diffs.md %})). In this example our pulsar node was called `p20` but that will be different for you.
```

Comment:
FAQs can be included as tip boxes easily as well:

`{% snippet topics/admin/faqs/diffs.md %}`
```suggestion
>    Here is an example of what those changes *might* look like in your Galaxy node.  In this example our pulsar node was called `p20` but that will be different for you.
> 
>    {% snippet topics/admin/faqs/diffs.md %} 
>
```

Reply:
yeah this one was my fault. thanks!